db/flush_job_test.cc

```diff
@@ -487,6 +507,137 @@ TEST_F(FlushJobTest, Snapshots) {
   job_context.Clean();
 }
 
+class FlushJobTimestampTest : public FlushJobTestBase {
+ public:
+  FlushJobTimestampTest()
+      : FlushJobTestBase(test::PerThreadDBPath("flush_job_ts_gc_test"),
+                         test::ComparatorWithU64Ts()) {}
+
+  void AddKeyValueToMemtable(MemTable* memtable, Slice key, uint64_t ts,
```

Comment:
`test::EncodeInt` (which is used to produce `key` when calling this method) returns an `std::string`; switching from `Slice` to `string` for `key` would eliminate some back-and-forth conversions between the two.